astronomer_starship/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "2.7.0"
+__version__ = "2.8.0"
```

Comment:
Do we want to release this a feature release **2.8.0** or rather as a patch release **2.7.1**. Basically functionality has not changed. But some might argue that a new UI is indeed a new feature.